docs/portal-contribute.md

```diff
@@ -142,6 +148,11 @@ developer-portal
 │       ├── dotnet
 │       ├── js
 |       └── python
+├── scripts
+│   ├── cip.ts
+│   ├── rust-library.ts
+│   ├── token-registry.ts
+│   └── *.md
```

Comment:
incidentally, what are `*.md` in this directory?  Is this a hook to add internal script documentation later? (don't see any markdown files there now)

Reply:
At the moment there are constants and reusable functions, but probably should be moved later to a separate folder.